clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -3383,6 +3383,18 @@ def err_callback_callee_is_variadic : Error<
   "'callback' attribute callee may not be variadic">;
 def err_callback_implicit_this_not_available : Error<
   "'callback' argument at position %0 references unavailable implicit 'this'">;
+
+def err_capture_by_attribute_multiple : Error<
+  "multiple 'lifetime_capture' attributes specified">;
+def err_capture_by_attribute_no_entity : Error<
+  "'lifetime_capture_by' attribute specifies no capturing entity">;
+def err_capture_by_implicit_this_not_available : Error<
+  "'lifetime_capture_by' argument references unavailable implicit 'this'">;
+def err_capture_by_attribute_argument_unknown : Error<
+  "'lifetime_capture_by' attribute argument %0 is not a known function parameter"
+  "; must be a function parameter of one of 'this', 'global' or 'unknown'">;
```

Comment:
>  a function parameter **of** one of 'this', 'global' or 'unknown'"

The `of` seems like a typo, I think it is `or`. Maybe just `must be a function parameter, this, 'global', or 'unknown'`

Reply:
Done.